PathKit/Sources/PathKit/PathKit.h

```diff
@@ -0,0 +1,18 @@
+//// Copyright © 2019 JABT Labs. All rights reserved.
```

Comment:
Is the double comment start coming from the template? We should fix that.

Reply:
done in previous PR